app/src/services/kleros.ts

```diff
@@ -29,12 +29,16 @@ class KlerosService {
     this.provider = provider
     this.tcrAddress = tcrAddress
 
-    this.badgeContract = new ethers.Contract(badgeContractAddress, klerosBadgeAbi, provider)
-    this.tokensViewContract = new ethers.Contract(tokensViewContractAddress, klerosTokensViewAbi, provider)
-    if (signerAddress) {
-      const signer = provider.getSigner()
-      this.badgeContract = this.badgeContract.connect(signer)
-      this.tokensViewContract = this.tokensViewContract.connect(signer)
+    const networkId = provider.network ? provider.network.chainId : null
```

Comment:
I changed this line to avoid an error. With TS 3.7 we could the use safe navigation operator.